server/index.js

```diff
@@ -67,6 +67,111 @@ app.get('/api/products/:productId', (req, res, next) => {
     });
 });
 
+app.get('/api/cart/', (req, res, next) => {
+  if (typeof req.session.cartId === 'undefined') {
+    res.json([]);
+  } else {
+    const sql = `select "c"."cartItemId",
+      "c"."price",
+      "p"."productId",
+      "p"."image",
+      "p"."name",
+      "p"."shortDescription"
+      from "cartItems" as "c"
+      join "products" as "p" using ("productId")
+      where "c"."cartId" = $1`;
+    const values = [req.session.cartId];
+    return db.query(sql, values)
+      .then(result => {
+        const cartItem = result.rows[0];
+        const newArr = [];
+        newArr.push(cartItem);
+        return res.json(newArr);
```

Comment:
There might be more than one item in the cart, just send all of the `result.rows`